src/main/java/uk/gov/hmcts/darts/common/entity/CourtCaseEntity.java

```diff
@@ -86,6 +95,16 @@ public class CourtCaseEntity extends CreatedModifiedBaseEntity {
     @Column(name = VERSION_LABEL, length = 32)
     private String legacyVersionLabel;
 
+    @Column(name = IS_DELETED)
+    private boolean isDeleted;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "deleted_by")
+    private UserAccountEntity deletedBy;
+
+    @Column(name = "deleted_ts")
+    private OffsetDateTime deletedTs;
```

Comment:
another consistency question - some fields are "Ts", some "Timestamp", some "DateTime"

Reply:
renamed to timestamp